lib/DDG/Spice/RandPOS.pm

```diff
@@ -0,0 +1,36 @@
+package DDG::Spice::RandPOS;
+# ABSTRACT: Returns a random word or list of words from the specified part of speech using Wordnik.
+
+use DDG::Spice;
+
+name 'Random Part of Speech Words';
+source 'Wordnik';
+description 'Returns one or more random words in the specified part of speech.';
+primary_example_queries 'random noun','example adjectives','adverb example';
+category 'language';
+topics 'words_and_games','everyday';
+code_url 'https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/RandPOS.pm';
+attribution github => ['https://github.com/SueSmith', 'SueSmith'],
+    web => ['http://benormal.info','Sue Smith'],
+    twitter => 'braindeadair';
+
+triggers start => 'random';
+triggers startend => 'example';
+
+spice from => '([^/]+)/?(?:([^/]+)/?(?:([^/]+)|)|)';
+spice to => 'http://api.wordnik.com/v4/words.json/randomWords?includePartOfSpeech=$1&minCorpusCount=10000&limit=$2&api_key={{ENV{DDG_SPICE_WORDNIK_APIKEY}}}&callback={{callback}}';
+
```

Comment:
Seeing as this is a "random" Spice, we don't want to cache the API response.

You'll need to add `spice proxy_cache_valid => '418 1d'` to this like the RandWord Spice does: https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/RandWord.pm#L18

Details: https://duck.co/duckduckhack/spice_advanced_backend#caching